src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThread.js

```diff
@@ -97,22 +123,29 @@ const AnnotationThread = ({
                         getMentionWithQuery={getMentionWithQuery}
                         handleCancel={handleCancel}
                         mentionSelectorContacts={mentionSelectorContacts}
-                        onAnnotationCreate={onAnnotationCreate}
+                        onAnnotationCreateEnd={handleAnnotationCreateEnd}
+                        onAnnotationCreateStart={handleAnnotationCreateStart}
                         onError={onError}
                         target={target}
                     />
                 ) : (
                     <AnnotationThreadContent
-                        api={api}
+                        annotation={annotation}
                         annotationId={annotationId}
```

Comment:
It seems we do not need `annotationId` anymore

Reply:
Correct, it's already removed from `AnnotationThreadContent` props.